hack/olm-registry/hypershift-template.yaml

```diff
@@ -0,0 +1,214 @@
+apiVersion: v1
+kind: Template
+metadata:
+  name: hypershift-dvo-selectorsyncset-template
+parameters:
+  - name: REGISTRY_IMG
+    required: true
+  - name: IMAGE_TAG
+    required: true
+  - name: IMAGE_DIGEST
+    required: true
+  - name: REPO_NAME
+    value: deployment-validation-operator
+    required: true
+  - name: DISPLAY_NAME
+    value: Deployment Validation Operator
+    required: true
+  - name: CHANNEL
+    value: "alpha"
+    displayName: OLM subscription channel
+    description: OLM subscription channel
+    required: true
+  - name: TARGET_ORG
+    displayName: Target organization
+    description: Target api.openshift.com organization to deploy to
+    required: true
+objects:
+  - apiVersion: hive.openshift.io/v1
+    kind: SelectorSyncSet
+    metadata:
+      annotations:
+        component-display-name: ${DISPLAY_NAME}
+        component-name: ${REPO_NAME}
+      labels:
+        managed.openshift.io/gitHash: ${IMAGE_TAG}
+        managed.openshift.io/gitRepoName: ${REPO_NAME}
+        managed.openshift.io/osd: "true"
+      name: deployment-validation-operator
+    spec:
+      clusterDeploymentSelector:
```

Comment:
MC cluster selector should be:
```yaml
      clusterDeploymentSelector:
        matchExpressions:
          - key: ext-hypershift.openshift.io/cluster-type
            operator: In
            values:
              - management-cluster
          - key: api.openshift.com/fedramp
            operator: NotIn
            values:
              - "true"
      resourceApplyMode: Sync
```

Reply:
Actually the selector should be:
```
      clusterDeploymentSelector:
        matchLabels:
          ext-hypershift.openshift.io/cluster-type: service-cluster
```
We need to deploy the policy, the placement rule and the binding rule on the service cluster. Those objects are then duplicated by the service cluster on each klusterlet namespace for each hosted cluster on each management cluster.